File: 283_Move_zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int next = 0;
        for (int number : nums) {
            if (number != 0)
                nums[next++] = number;
        }
        for (int i=next; i<nums.length; i++) {
            nums[i] = 0;
        }
    }
}

File: Set1_1000/283_Move_zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        i = 0
        for element in range(len(nums)):
            if nums[element] != 0:
                nums[i], nums[element] = nums[element], nums[i]
                i = i + 1
        
